common/src/test/java/tachyon/util/io/PathUtilsTest.java

```diff
@@ -16,13 +16,43 @@
 package tachyon.util.io;
 
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import tachyon.Constants;
+import tachyon.thrift.InvalidPathException;
 
 public class PathUtilsTest {
+  @Rule
+  public final ExpectedException mException = ExpectedException.none();
+
+  @Test
+  public void getPathComponentsTestSuccess() {
+    try {
+      Assert.assertArrayEquals(new String[] {""}, PathUtils.getPathComponents("/"));
+      Assert.assertArrayEquals(new String[] {"", "bar"}, PathUtils.getPathComponents("/bar"));
+      Assert.assertArrayEquals(new String[] {"", "foo", "bar"},
+          PathUtils.getPathComponents("/foo/bar"));
+      Assert.assertArrayEquals(new String[] {"", "foo", "bar"},
+          PathUtils.getPathComponents("/foo/bar/"));
+      Assert.assertArrayEquals(new String[] {"", "bar"},
+          PathUtils.getPathComponents("/foo/../bar"));
+      Assert.assertArrayEquals(new String[] {"", "foo", "bar", "a", "b", "c"},
+          PathUtils.getPathComponents("/foo//bar/a/b/c"));
+    } catch (InvalidPathException e) {
+      Assert.fail();
+    }
+  }
+
+  @Test
+  public void getPathComponentsTestException() throws InvalidPathException {
```

Comment:
Tachyon convention is to name testcase suffixed with `Test` :)
Same comment applies to above testcase.